### DISCUSSION TITLE
Ladder not long enough to get into a module

### DISCUSSION THREAD

**Discussion by Vladislav762** [FakeFishGames/Barotrauma#14896] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [x] My issue happened while using mods.

### What happened?

i can't get up to the npc quest bot, because the ladder is not tall enough
![bara](https://github.com/user-attachments/assets/a35ddd6e-ba5f-43ce-9efd-ed22b262036c)

### Reproduction steps

started single mode sailed to the outpost, wanted to take quest, cant reach it because of the ladder
![bara](https://github.com/user-attachments/assets/3bd19cfe-406e-4ab1-933d-b21d5266ed2a)

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.6.17.0 (Unto the Breach Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! 

I unfortunately could not reproduce the issue - there seems to be a pre-placed ladder in this module (AdminModule_02R), so it should not be necessary for the ladder to extend from the vertical shaft up to the module.
![image](https://github.com/user-attachments/assets/bb04a2e7-a991-410a-b5d0-ab00be777123)

You ticked the checkbox that says you're using mods - what kind of mods? Could it be that some of them modify the outpost modules somehow?

**Reply by Vladislav762** [reply]
dynamic europa
autodoors for outpost
shiprecks extended lua for baro
![image](https://github.com/user-attachments/assets/828b5ec9-7dd1-40ed-b28d-c02e2992be97)
![image](https://github.com/user-attachments/assets/bfa3bc4e-c0fe-496a-ad1c-a3483628d02f)

**Reply by NotWendy** [reply]
Dynamic Europa uses its own modules instead of vanilla ones. Some of those custom modules are _edited_ vanilla modules. And autodoors for outposts might even clash with it. So the problem was likely caused by one of those two.

**Reply by Vladislav762** [reply]
Ok, ill try
thanks a lot

**Comment by NilanthAnimosus** [comment]
This is an issue with the mod "Auto-Doors for Outposts"

![image](https://github.com/user-attachments/assets/5c04831d-c404-49f5-916c-f57f7d5843c6)

It has a ladder linked to the top hatch only, which the behaviour of this changed at some point where linking a ladder can allow its removal with unused hatches when not linking to a module. In this case, it is linked to the top hatch which is unused, so the ladder is removed and is working as intended. (And the mod needs to make changes to the module).

Closing as its not a bug but a mod issue.
